mlir/utils/generate-test-checks.py

```diff
@@ -236,9 +236,12 @@ def process_attribute_references(line, attribute_namer):
 # Pre-process a line of input to remove any character sequences that will be
 # problematic with FileCheck.
 def preprocess_line(line):
+    # If input line has `{{` i.e. in a StrAttr with serialized proto.
```

Comment:
I would not mention serialized protos in the comment, the important thing here is just escaping {{ which correspond to regex in file check.

Reply:
Good point - Changed to be more consistent with the other comments in this function:

```
    # Replace any `{{` with escaped replacements. `{{` corresponds to regex
    # checks in FileCheck.
```